### PR TITLE
Add Visual Studio 17 2022 to CI tests

### PR DESCRIPTION
**Pull request by BenjaminW3**

### CODE REVIEW

**Review by bernhardmgruber — DISMISSED** [review]

**Comment by bernhardmgruber** [comment]
`fatal error C1060: compiler is out of heap space [D:\a\alpaka\alpaka\build\test\unit\mem\buf\memBufTest.vcxproj]`

I wonder whether you can refactor that test to use less heap. Maybe split the translation unit into multiple.

**Comment by BenjaminW3** [comment]
I am not sure if we are using the 64 bit compiler at all, or if it is by default using the 32 bit compiler build.
I will try if we can call `vcvars64.bat` to use the 64 bit build before compilation.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -230,7 +237,7 @@ jobs:
         # icpc
         - name: linux_icpc_release
           os: ubuntu-latest
-          env: {CXX: icpc,    CC: icc,                                 ALPAKA_CI_STDLIB: libstdc++, CMAKE_BUILD_TYPE: Release, ALPAKA_CI_BOOST_BRANCH: boost-1.75.0, ALPAKA_CI_CMAKE_VER: 3.19.2, OMP_NUM_THREADS: 2, ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:20.04",                                                                                                                                         ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF, ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE: OFF}
+          env: {CXX: icpc,    CC: icc,                                 ALPAKA_CI_STDLIB: libstdc++, CMAKE_BUILD_TYPE: Release, ALPAKA_CI_BOOST_BRANCH: boost-1.75.0, ALPAKA_CI_CMAKE_VER: 3.19.2, OMP_NUM_THREADS: 2, ALPAKA_CI_DOCKER_BASE_IMAGE_NAME: "ubuntu:20.04", ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: OFF, ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE: OFF}
```
Comment (j-stephan) [inline .github/workflows/ci.yml:240]: Why the whitespace change here?

Comment (BenjaminW3) [reply .github/workflows/ci.yml:240]: Yes, because this line was not aligned with the other ones above.

**Comment by BenjaminW3** [comment]
Either we had luck, or the call to `vcvars64.bat` really helped.
As one of the linux builds failed, we have to give it another try either way.

**Comment by BenjaminW3** [comment]
Ready for approvals ;-)

**Review by j-stephan — APPROVED** [review]

**Pull request merged**